### PR TITLE
Fixed small ExpandableBlock padding

### PR DESCRIPTION
**Pull request by DanEastBentley**
Fixed size of small ExpandableBlock from iTwinUI-react by setting the padding and icon top-margin to 5px instead of 4px.
Cannot fix the lack of left, top, right border without changes to iTwinUI-css.

### CODE REVIEW

**Review by bsteinbk — APPROVED** [review]

File: ui/core/src/ui-core/style/itwinui-overrides.scss [hunk 1]
```diff
@@ -46,14 +46,16 @@
...
   .iui-expandable-block {
     > .iui-header {
-      padding: $iui-xs $iui-xs !important;
+      padding: $small-space $small-space !important;
 
       > .iui-icon {
         width: $uicore-icons-x-small !important;
         height: $uicore-icons-x-small !important;
-        margin-top: $iui-xs !important;
+        margin-top: $small-space !important;
```
Comment (roluk) [inline ui/core/src/ui-core/style/itwinui-overrides.scss:58]: Vertical alignment looks better now, but the chevron still looks slightly off horizontally.

firefox_CYbgiG58UI

**Review by roluk — CHANGES_REQUESTED** [review]

File: ui/core/src/ui-core/style/itwinui-overrides.scss (hunk 1 above)
Comment (DanEastBentley) [reply ui/core/src/ui-core/style/itwinui-overrides.scss:58]: Does it look different than these: https://itwin.github.io/iTwinUI-react/?path=/docs/core-expandableblock--basic ? Please suggest a satisfactory margin-left.

Comment (roluk) [reply ui/core/src/ui-core/style/itwinui-overrides.scss:58]: iTwinUI-react ExpandableBlock has 12px padding on the header and 12px right margin on the chevron. To achieve similar balance at our current 5px header padding, I'd add 3px to chevron's left margin and set its right margin to 8px.

Comment (DanEastBentley) [reply ui/core/src/ui-core/style/itwinui-overrides.scss]: I pushed a fix

Comment (roluk) [reply ui/core/src/ui-core/style/itwinui-overrides.scss]: Thanks

**Review by roluk — APPROVED** [review]

**Pull request merged**